.github/workflows/CI.yml

```diff
@@ -39,8 +39,10 @@ jobs:
       - uses: actions/setup-python@v2
         with:
           python-version: 3.7
+      - run: npm ci   
```

Comment:
Ahh I see. This won't work. 

Look above in the build job - it's installing node - but this test job is not. 
Hence why stylelint is not available.

Reply:
It worked!

Reply:
Awesome. The CI is running but the linting is picking up heaps of issues - are you ok to fix the issues also.

Reply:
Now we only have to fix every line of scss written for this project....
Please don't merge this one!😢

Reply:
All good. I was surprised there were no linting errors originally haha. 
It's not a huge list - reach out if you need a hand. Make sure you can run the linting in your editor as it will make debugging much easier.

Reply:
It is a huge list!😅